terraform/modules/swipe-sfn-batch-queue/main.tf

```diff
@@ -68,8 +68,13 @@ resource "aws_launch_template" "swipe_batch_main" {
   # not recognize this change. We bind the launch template name to user data contents here, so any changes to user data
   # will cause the whole launch template to be replaced, forcing the compute environment to pick up the changes.
   name      = "${var.app_name}-batch-main-${local.launch_template_user_data_hash}"
-  user_data = base64encode(local.launch_template_user_data)
   tags      = var.tags
+  user_data = base64encode(concat(var.user-data-parts,
+      {
+        filename = local.launch_template_user_data_file,
+        content_type = "multipart/mixed",
+        content = file(local.launch_template_user_data_file)
+      }))
```

Comment:
The existing user data is already a multipart, can we mix a multipart file with other files like this? I think we may need to pull apart the pieces of the existing user data and then build them into a multipart with the other pieces.

Reply:
https://github.com/chanzuckerberg/shared-infra/pull/5153/files#diff-03f83f2902ab11032bdcb34beefa58f27ead92a7f43288f67ee2db51e859ef65R125 I pulled apart the pieces here, so my understanding is that this results in a concat of all the parts separately